lldb/tools/lldb-dap/OutputRedirector.cpp

```diff
@@ -38,7 +38,7 @@ Expected<int> OutputRedirector::GetWriteFileDescriptor() {
   return m_fd;
 }
 
-Error OutputRedirector::RedirectTo(std::FILE *file_override,
+Error OutputRedirector::RedirectTo(std::FILE *fileOverride,
```

Comment:
But now this doesn't conform to lldb's coding style :( Why not change line 59-60 to `file_override`?

Reply:
Sorry for the churn, I saw I broke the CI job so I was trying to fix it to unblock that.

Created #130202 to conform to lldb's coding style.

Reply:
Yeah no worries. Thanks for the quick turnaround!